data/102/030/739/102030739.geojson

```diff
@@ -63,10 +63,12 @@
         "Belgaum"
     ],
     "name:eng_x_preferred":[
-        "Belgaum"
+        "Belagavi"
     ],
     "name:eng_x_variant":[
-        "IXG"
+        "IXG",
+        "Belgaum",
```

Comment:
It seems like this is a conventional historic English name (while the modern Indian English name is the one now in the preferred slot)... Should we x-list it in the historic name variant in that WOF slot, too?

Reply:
Yeah, good call.. I've opened https://github.com/whosonfirst-data/whosonfirst-data-admin-in/pull/111 to x-list the name in the historical property, too.. closing this PR.